task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -66,7 +65,7 @@ def retry_loop(self):
             while not self.src_queue.empty():
                 e = self.src_queue.get()
 
-                if e.kind == 'control':
+                if e.kind != 'task':
```

Comment:
they are, but we don't know what kind of events we might have in future and retrying executor is probably only ever concerned with task events